challenge_7/python/wost/find_missing_number.py

```diff
@@ -0,0 +1,14 @@
+def find_biggest(a):
+	biggest = a[0]
+	for i in a:
+		if i >= biggest:
+			biggest = i
+	return biggest
+
+def find_missing_number(c):
+	b = find_biggest(c)
```

Comment:
Hm, no. I'd have to find the smallest number as well.
I'll update this when I find out about this! :)